streamer/src/nonblocking/quic.rs

```diff
@@ -26,6 +26,7 @@ use {
 };
 
 const QUIC_TOTAL_STAKED_CONCURRENT_STREAMS: f64 = 100_000f64;
+const WAIT_FOR_STREAM_TIMEOUT_MS: u64 = 1000;
```

Comment:
I think this timeout is too long because it means that we are only guaranteed to check if a connection has been pruned once per second. Is there any harm in making it shorter so that pruning gets more effective?

Reply:
We can probably reduce it. We should consider maximum round trip delay we want to tolerate for the handshake. It'll still be in terms of hundred of millisec to be on a safer side.

Reply:
What makes you concerned about the handshake time? My understanding of the quic server is that slow connections don't block new connections at all because they get pruned away. The only overhead is polling the handshake future which is pretty lightweight.